tests/php/Security/MemberTest.php

```diff
@@ -1605,7 +1606,9 @@ public function testCustomMemberValidator()
         $this->assertFalse($fail, 'Missing FirstName');
 
         $ext = new MemberTest\ValidatorExtension();
-        $ext->updateValidator($validator);
+        $method = new ReflectionMethod(MemberTest\ValidatorExtension::class, 'updateValidator');
+        $method->setAccessible(true);
+        $method->invokeArgs($ext, [&$validator]);
```

Comment:
```suggestion
        $method->invokeArgs($ext, [$validator]);
```
Don't need to pass objects by reference - and if you did it should be handled at the receiving end.

Reply:
Updated, removed the pass by reference bit on the receiving end which shouldn't be there because an object is being passed